_includes/nav.html

```diff
@@ -1,87 +1,147 @@
 <!-- Header for screens with width > 1260px and width <= 767px -->
-<header class="large" id="firstHeader" >
+<header class="large" id="firstHeader">
   <div class="row">
     <div class="container">
-      <nav class="navbar navbar-default navbar-expand-lg fixed-top align-items-center d-flex justify-content-between pb-0 pt-0">
-        <div class="navbar-header">
-          <div id="navh" class="logo1">
-            <a href="{{ site.baseurl }}/">{% include logo.svg %}</a>
-          </div>
-          <div id="navh2" >
-            <a class="navbar-brand" href="{{ site.baseurl }}/">
-              <img src="{{ site.baseurl }}/assets/logoOpt.png" alt="Logo">
-            </a>
-          </div>
-        </div>	
+      <nav class="navbar navbar-default navbar-expand-lg fixed-top align-items-center d-flex justify-content-between     pb-0  pt-0">
+        <a class="navbar-brand" href="{{ site.baseurl }}/">
+          <img src="{{ site.baseurl }}/assets/logoOpt.png" alt="Sugar Labs logo" height="90" width="270">
```

Comment:
Why change this from `{{ site.baseurl }}/">{% include logo.svg %}` ?

The color change feature is nice, so it would be nice to keep it.